packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -38,11 +38,6 @@ class EmptyScreen extends Screen {
     Octicons.getIconData('pulse'),
   );
 
-  static final EmptyScreen performance = EmptyScreen(
```

Comment:
why did we remove the performance screen?

Reply:
because we now have a real performance screen.